Brewfile

```diff
@@ -1,4 +1,9 @@
 brew 'python@2'
+brew 'yarn'
 brew 'libxmlsec1'
 brew 'openssl'
-brew 'chromedriver'
+brew 'redis', restart_service: true
```

Comment:
Can we lock this to a specific version of redis like we can for Postgres? I ask becuase things will get updated out from under people otherwise which has bad news written all over it.

Reply:
Our docs [here](https://docs.sentry.io/internal/environment/) don't specify a version to pin, and [these docs](https://docs.sentry.io/server/installation/) say `the minimum version requirement is 2.8.9, but 2.8.18, 3.0, or newer are recommended` and also suggests using `redis:3.2-alpine`. Furthermore, the latest stable redis in Homebrew is `4.0.9` and all tests pass on my local machine with it.

Reply:
What exactly should we be pinning? Whatever redis version is being used in Travis CI? Is there a source of truth for Sentry's "system" dependencies (everything that isn't a Python or Node/Yarn package)? Because our documentation is vague and also conflicting (see below).

Reply:
Yup, these are complicated because we use various different versions in production. Just depending on if the cluster is new, etc. So I don't think it's that big of a deal. My only concern is how Brew always tracks latest and things will always upgrade to the newest no matter what.

So one day, Redis 5 or something will come out, and this _may_ cause problems, in which case, how do people roll back to an older version? We can always address this later, but was just asking if we could pin.

Reply:
I'd say, if it's able to be pinned, picking latest 4.0.x would be safest, but if not, we just deal with it when it inevitably happens.